src/test/resources/alma/(DE-605)HT015011399.json

```diff
@@ -51,6 +51,10 @@
     "label" : "Culturegraph Ressource"
   } ],
   "thesisInformation" : [ "Dortmund, Univ., Diss., 2007" ],
+  "related" : [ {
+    "label" : "Strukturelle und biochemische Einblicke in Mechanismen des Nukleotidaustauschs von Rab-Proteinen",
+    "note" : "Erscheint auch als Druckausg. u.d.T."
```

Comment:
Is the subfield "t" always a reiteration of the main resources title? It does not seem very senseful then and we might better dump that for it's just bloating and obfuscating the metadata IMO.

Reply:
No $t states the title of the refrenced ressource. e.g. if it is a translation it states the name of the original work.